recipes/flojoy/meta.yaml

```diff
@@ -0,0 +1,81 @@
+{% set name = "flojoy" %}
+{% set version = "0.1.15" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/flojoy-{{ version }}.tar.gz
+  sha256: 3ce686dc94b5be85c9f75021bc122d0eca92c9cf0d332f8d68d9653c336b1e9e
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+
+requirements:
+  host:
+    - python >=3.10,<3.11
+    - poetry-core
+    - pip
+  run:
+    - python >=3.10.0,<3.11.0
+    - pandas >=2.0.0,<3.0.0
+    - numpy >=1.24.0,<2.0.0
+    - plotly >=5.12.0,<6.0.0
+    - pillow >=10.0.0,<11.0.0
+    - python-dotenv >=1.0.0,<2.0.0
+    - pyyaml >=6.0.0,<7.0.0
+    - requests >=2.29.0,<3.0.0
+    - keyring >=24.2.0,<25.0.0
+    - python-box >=7.0.0,<8.0.0
+    - pydantic >=2.0.0,<3.0.0
+    - huggingface_hub >=0.16.0,<0.17.0
+    - cloudpickle >=2.2.1,<3.0.0
+    - networkx >=3.0.0,<4.0.0
+    - scipy >=1.10.0,<2.0.0
+    - matplotlib-base >=3.6.0,<4.0.0
+    - portalocker >=2.7.0,<3.0.0
+    - qcodes >=0.39.1,<0.40.0
```

Comment:
The pinning here might be quite strict here.

Reply:
I would only keep the pin on the major version provided that packages use semantic versioning. e.g. for scipy this would be:

```yaml 
    - scipy <2.0.0
```

Reply:
Changing `pyproject.toml` should be sufficient.